examples/client-mithril-stake-distribution/README.md

```diff
@@ -0,0 +1,15 @@
+# Mithril client library example: Mithril stake distribution
+
+## Build and run the example
+
+```bash
+# Build
+cargo build -p client-mithril-stake-distribution
```

Comment:
You can say that after going into the crate directory, build is done by `cargo build` and run by `cargo run`